tests/test_component.py

```diff
@@ -55,7 +57,7 @@ def test_component_repl_demo():
 
     assert "GenericDependency" in agent_comp.dependencies.keys()
     inst_dep_obj = agent_comp.dependencies["GenericDependency"].get_object()
-    assert type(inst_dep_obj) == GenericDependency
+    assert inst_dep_obj.__class__.__name__ == "GenericDependency"
```

Comment:
Aren't these equivalent? Why the switch to this alternate syntax? Or maybe I'm missing something.